build/misc.js

```diff
@@ -77,30 +87,22 @@ function getLocals() {
              * @param  {Number} code
              * @return {String}
              */
-            getStatusMessage(code) {
+            getStatusMessage (code) {
                 return http.STATUS_CODES[code];
-            }
+            },
         },
     };
-
-    let common;
-    try {
-        out.permissions = require('@mcph/beam-common').permissions;
```

Comment:
This was done so that developers who do not have access to our private npm org can still contribute to the developer site